custom-metrics-stackdriver-adapter/pkg/adapter/provider/provider.go

```diff
@@ -95,7 +95,11 @@ func (p *StackdriverProvider) GetRootScopedMetricByName(groupResource schema.Gro
 	if err != nil {
 		return nil, err
 	}
-	stackdriverRequest, err := p.translator.GetSDReqForNodes(&v1.NodeList{Items: []v1.Node{*matchingNode}}, metricName)
+	metricKind, err := p.translator.GetMetricKind(p.translator.config.MetricsPrefix + "/" + metricName)
```

Comment:
Can you do that metric name building in GetMetricKind argument? It would be way more natural if it was prepending the prefix itself.

Reply:
Done